nixos/modules/services/networking/firefox/sync-server.nix

```diff
@@ -65,6 +70,18 @@ in
         '';
       };
 
+      user = mkOption {
+        type = types.str;
+        default = "syncserver";
+        description = "User account under which syncserver runs.";
+      };
```

Comment:
Indeed I don't see good reasons for changing this, I just assumed it was standard procedure to have the user configurable. I'll remove the option for now and if someone needs it they can open an issue for it.